webdev-day2.md

```diff
@@ -0,0 +1,55 @@
+# CSS - Flexible Box, Position, Grid, Media query
+
+## Flexible Box
+The Flexible Box Module, usually referred to as flexbox, was designed as a one-dimensional layout model, and as a method that could offer space distribution between items in an interface and powerful alignment capabilities.
+
+The main idea behind the flex layout is to give the container the ability to alter its items’ width/height (and order) to best fill the available space (mostly to accommodate to all kind of display devices and screen sizes). A flex container expands items to fill available free space or shrinks them to prevent overflow.
+
+<details>
+<summary>Basics and terminology</summary>

```

Comment:
While the idea of adding <summary> is neat, it breaks inner alignment. Resorting to using a H3 (`###`) should be enough.